File: python/beginner/1013.py
# greatestNumber = max(map(int, input().split()))
[a, b, c] = map(int, input().split())
abGreatest = (a + b + abs(a - b)) / 2
greatestNumber = (abGreatest + c + abs(abGreatest - c)) / 2

print(f"{greatestNumber:.0f} eh o maior")

File: c/beginner/1013.c
#include <stdio.h>
#include <stdlib.h>

int main(void) {
  int a, b, c;

  scanf("%d %d %d", &a, &b, &c);

  int abGreatest = (a + b + abs(a - b)) / 2;
  printf("%d eh o maior\n", (abGreatest + c + abs(abGreatest - c)) / 2);

  return 0;
}
